docs/HelpWantedIssueSummaryCommentTemplate.md

```diff
@@ -0,0 +1,31 @@
+<!--
+
+**IMPORTANT**: As part of posting this comment, please make sure that:
+- the issue is properly labeled with both `area-` and `feature-` labels.
+- one of the `Complexity: ` labels is applied to the issue [Simple | Medium | Hard]
```

Comment:
Just to clarify, will the `good first issue` be superseded by this `simple` label?

Reply:
I think so. I don't have it explicitly called it out, but that's the plan.

Reply:
Ah gotcha. Just wanted to check as that `good first issue` label is specifically called out in [CONTRIBUTING.md](https://github.com/dotnet/aspnetcore/blob/main/CONTRIBUTING.md#finding-an-issue-to-work-on).

Reply:
Glad that you've brought this up. Perhaps I should (for now) leave it as is, and call it out that some "simple" issues may be a good fit for first time contributors. Will add a note about this.